app.js

```diff
@@ -82,5 +83,16 @@ function encryptSentence(inputSentence) {
 }
 
 function letterCheck(clickedLetter) {
-  alert(clickedLetter);
+  state.roundCounter++;
+  const letterPositions = charPositions(clickedLetter, state.proverb);
+  let currentDisplayedProverb = state.displayedProverb;
+  letterPositions.forEach(item => currentDisplayedProverb = currentDisplayedProverb.substr(0, item) + clickedLetter + currentDisplayedProverb.substr(item + 1));
```

Comment:
if you are skipping `{ }` in arrow function you are saying that you want to return something. At first, there is no variable to catch the result of this expression, but also there is no chance for that, because forEach returns undefined. 

So, it will be more readable here:
```
letterPositions.forEach(item => {
.....
});
```